### ISSUE TITLE
Project Feedback

### ISSUE THREAD

**Issue by derpachu** [derpachu/MI-449-SS17-740-css-spacing-and-layout-T7bJIb#1]
@egillespie Can you take a look at this? It's [hosted here](https://tangjos1-kapm87bvfi5cut7bjib.surge.sh/) and meets the following criteria:

- [x] Demonstrate an understanding of the CSS we've learned so far in the class
- [x] Use Flexbox to create the layout of your page
- [x] Use `margin` and `padding` to create space in and between elements
- [x] Use display `block` and `inline-block` properly
- [x] Have fun and do your best to make it pretty, the result should be something you would show to a future employer

let me know if I should add more

**Comment by egillespie** [comment]
Hi @derpachu, I'm wondering if this site is something you would feel comfortable referring a potential employer to so they can learn more about you or see the quality of your work?

Here are some things that make me ask this question:

* The "guy1", "guy2", etc. references seem out of place (perhaps replace it with "References available upon request").
* Putting myself in the shoes of a non-technical hiring manager or recruiter, I can't tell much about your skills by looking at the site. I think a link or HTML version of your [resume](https://derpachu.github.io/info1.html), list of professional skills, and job interests would be great.
* I expected the "Professionalism", "Consistancy", "Leadership" boxes to be links. Also, "consistency" is spelled wrong.

**Comment by derpachu** [comment]
Yeah I wasn't sure how high fidelity you wanted the project to be. I'm not sure if the surge uploaded the resume correctly as it isn't coming up. If it isn't one of those errors that require me just to wait for it to load let me know.

**Comment by egillespie** [comment]
A quick peek at Chrome DevTools shows that the PDF did not upload to Surge:

![image](https://cloud.githubusercontent.com/assets/5572255/24664363/787cf040-1928-11e7-8b5d-90551415173b.png)

While getting your PDF uploaded, do you mind also doing the following:

1. Format your files so they pass linting.
2. Reduce the size of your background image. It's 17 MB and loads very slowly.
3. Replace `<1>` in your flexbox styles with a valid number. You may need to add additional styles to make your links center appropriately when you shrink the width of the browser window

I like your use of `display: block` and `display: inline-block`. Clever!

---

After you’ve made your changes and pushed them to GitHub and your hosted site, give it a once-over to make sure it looks right, then comment back here and I’ll take another look.

Thanks! 🐭

**Comment by derpachu** [comment]
just changed the resume to be an image instead. Wasn't sure which links you were referring to so hopefully what I did was what you wanted.

**Comment by egillespie** [comment]
This is much better, thanks! :shipit:
